cmd/stack_package.go

```diff
@@ -251,7 +253,7 @@ The packaging process builds the stack image, generates the "tar.gz" archive fil
 				return errors.Errorf("Error applying templating: %v", err)
 			}
 
-			// tag with the full version then mojorminor, major, and latest
+			// tag with the full version then majorminor, major, and latest
```

Comment:
We will need to tag the image with `MAJOR.MINOR.PATCH`, in order for the metadata to be accurate. 
We could add the flag discussed in #867 as part of this PR?
Another suggestion could be to add a `semver[majorminorpatch]` the same way we have `semver[majorminor]`

Reply:
Thanks for taking a look - as we discussed the major.minor.patch level is tagged on line 256 (`buildImage` variable).